sk/static/nyaablue.user.js

```diff
@@ -39,22 +40,49 @@ async function setViewBlue () {
 
   const { items } = await torrentsResponse.json()
   if (items[0]?.infoHash) {
-    element?.classList.add('panel-info')
+    element?.classList.add(items[0]?.isBest ? 'panel-info' : 'panel-info-alt')
+  }
+  if (items[0]?.id) {
+    const collectionResponse = await seadexEntryEndpoint('', {
+      filter: 'trs~"' + items[0]?.id + '"'
+    })
+
+    const entries = await collectionResponse.json()
+
+    if (entries.items?.length !== 0) {
+      const report = document.querySelector('body > div > div:nth-child(1) > div.panel-footer.clearfix > button')
+
+      document.head.insertAdjacentHTML('beforeend', '<style id="css_blue" type="text/css">button.btn-seadex {margin-right: 5px; color: #fff; background-color: #247fcc; border-color: #247fcc;} button.btn-seadex:hover {margin-right: 5px; color: #fff; background-color: #19578b; border-color: #19578b;} </style>')
```

Comment:
shouldnt that go [here?](https://github.com/ThaUnknown/releases-moe/pull/40/files#diff-ec9a791432ddc2e19aa339916a454f4b1e50cb674ec83e888a27c81f66343673R102)

Reply:
No point in adding it if it doesn't need to.